From 48657d5cca1d39c1d42cddc767fbc45a756a1edf Mon Sep 17 00:00:00 2001
Message-Id: <48657d5cca1d39c1d42cddc767fbc45a756a1edf.1425109419.git.chang-joon.lee@intel.com>
In-Reply-To: <ce15578fa48a21eff276ee8163d395e39785921c.1425109418.git.chang-joon.lee@intel.com>
References: <ce15578fa48a21eff276ee8163d395e39785921c.1425109418.git.chang-joon.lee@intel.com>
From: Deepak S <deepak.s@intel.com>
Date: Mon, 23 Feb 2015 04:07:37 -0500
Subject: [PATCH 2/4] SQUASHME! [VPG]: drm/i915: Remove wrong power gating in
 resume.

We already power gate the display in i915_pm_suspend, remove the wrong power gate call in pm_resume.

Squash Patch to:
commit 93d6a0ef2a144784b8623748522a1700663c002d
Author: Deepak S <deepak.s@intel.com>
Date:   Sat Jan 31 13:45:11 2015 +0530

    FOR_UPSTREAM [VPG]: drm/i915/vlv: Perform modeset based on DPMS
    state during resume

Issue: GMINL-6360
Change-Id: I8c6d33e6a6b2585a09899b3252ad345d52065530
Signed-off-by: Deepak S <deepak.s@intel.com>
---
 drivers/gpu/drm/i915/i915_drv.c |    8 --------
 1 file changed, 8 deletions(-)

diff --git a/drivers/gpu/drm/i915/i915_drv.c b/drivers/gpu/drm/i915/i915_drv.c
index 4f74afe..33daba5 100644
--- a/drivers/gpu/drm/i915/i915_drv.c
+++ b/drivers/gpu/drm/i915/i915_drv.c
@@ -812,14 +812,6 @@ static int __i915_drm_thaw(struct drm_device *dev, bool restore_gtt_mappings)
 	dev_priv->modeset_restore = MODESET_DONE;
 	mutex_unlock(&dev_priv->modeset_restore_lock);
 
-	/*
-	 * VLV has a special case and we need to avoid the display going to D0
-	 * until we get suspend.
-	 * */
-	if ((!IS_VALLEYVIEW(dev) || IS_CHERRYVIEW(dev))
-			&& display_is_on(dev))
-		intel_display_set_init_power(dev_priv, false);
-
 	sysfs_notify(&dev->primary->kdev->kobj, NULL, "thaw");
 
 	return 0;
-- 
1.7.9.5

